gateway-sp-comms/src/single_sp/update.rs

```diff
@@ -257,57 +258,137 @@ fn read_auxi_check_from_tlvc(data: &[u8]) -> Result<[u8; 32], UpdateError> {
     })
 }
 
+/// Isolate extraction of bootleby from old-format archives.
+/// TODO: Purge old-format archives and delete this code.
```

Comment:
Is this `TODO` to be addressed on this PR, or at some point in the future? (I don't know off hand when we switched away from `old-format`)

Reply:
I clarified the text: "When old-format archives are eliminated from customer racks and spares inventory, then this code can be removed."